packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -333,7 +333,7 @@ export function parseEnvironment(values: RAW_ENVIRONMENT): ENVIRONMENT {
  * populating default values.
  */
 export function getEnvWithoutDefaults(): ENVIRONMENT {
-  return typeof process !== 'undefined'
+  return typeof process !== 'undefined' && process && process.env
```

Comment:
```suggestion
  return process && process.env
```

Reply:
typeof for process is required here to ensure that process indeed exists. It is not necessary that process would exist always.